RxCocoa/iOS/UIViewController+Rx.swift

```diff
@@ -14,13 +14,40 @@
 #endif
 
     extension Reactive where Base: UIViewController {
-
+        
         /// Bindable sink for `title`.
         public var title: UIBindingObserver<Base, String> {
             return UIBindingObserver(UIElement: self.base) { viewController, title in
                 viewController.title = title
             }
         }
-    
+        
+        public var viewDidLoad: ControlEvent<[Any]> {
+            return ControlEvent(events: sentMessage(#selector(UIViewController.viewDidLoad)))
+        }
+        
+        public var viewWillAppear: ControlEvent<[Any]> {
+            return ControlEvent(events: sentMessage(#selector(UIViewController.viewWillAppear(_:))))
+        }
+        
+        public var viewDidAppear: ControlEvent<[Any]> {
+            return ControlEvent(events: sentMessage(#selector(UIViewController.viewDidAppear(_:))))
+        }
+        
+        public var viewWillDisappear: ControlEvent<[Any]> {
+            return ControlEvent(events: sentMessage(#selector(UIViewController.viewWillDisappear(_:))))
+        }
+        
+        public var viewDidDisapppear: ControlEvent<[Any]> {
```

Comment:
We should return the proper `Element` types here. For example, this one is `Bool`. And `viewDidLoad` is `Void`.